buildkite/download/action.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
What do you think about the name `buildkite/download-artifacts`?

Reply:
I guess it could be renamed to `path`?

```yaml
- uses: elastic/oblt-actions/buildkite/download-artifacts@v1
        with:
          path: "artifacts.tar"
          build: ${{ steps.buildkite-run.outputs.build }}
          pipeline: "my-super-pipeline"
          token: ${{ secrets.BUILDKITE_TOKEN }}
```

Only "download" sounds so ambiguous. 🤷 

also if you plan to create an action called `buildkite/log` then I think `buildkite/artifacts` sounds like a better fit.

but generally.. I think `buildkite/download-artifacts` is explicit and doesn't leave ambiguity

Reply:
Same pattern in https://github.com/actions/download-artifact, so I'll apply your suggestion, thanks for this :)

Reply:
see [`5e8e6e6` (#18)](https://github.com/elastic/oblt-actions/pull/18/commits/5e8e6e6d8614da8adaf795511bf675eef47e542e)